pixi/src/checks/PageExperienceCheck.js

```diff
@@ -81,7 +80,7 @@ export default class PageExperienceCheck {
       },
     };
 
-    return reports;
+    return [null, report];
```

Comment:
Returning null as first value feels wrong.

Reply:
@robinvanopstal, this goes with: https://github.com/ampproject/amp.dev/pull/4314#discussion_r473010712 - instead of returning a tuple make alle `*.createReportData` methods return `{error: *, data: *}`